src/main/java/org/opensearch/sdk/ExtensionsRunner.java

```diff
@@ -188,7 +223,7 @@ public void startTransportService(TransportService transportService) {
             false,
             false,
             IndicesModuleRequest::new,
-            ((request, channel, task) -> channel.sendResponse(handleIndicesModuleRequest(request)))
+            ((request, channel, task) -> channel.sendResponse(handleIndicesModuleRequest(request, transportService)))
```

Comment:
We'll need to pass transportService here for the API and also for the tests. Since instance variable is removed